main/xrdp/APKBUILD

```diff
@@ -1,15 +1,13 @@
 # Contributor: Alan Lacerda <alacerda@alpinelinux.org>
 # Maintainer: Alan Lacerda <alacerda@alpinelinux.org>
 pkgname=xrdp
-pkgver=0.9.9
+pkgver=0.9.10
 pkgrel=0
 pkgdesc="An open source RDP server"
-url="http://www.xrdp.org/"
+url="http://xrdp.org/"
```

Comment:
`xrdp.org` and `www.xrdp.org` actually have different IP address.  The former doesn't load here, the latter is a github page. Besides: `www.xrdp.org` seems to support HTTPS.

Reply:
Fixed, thanks for catching